### PR TITLE
gitk: add page

### PR DESCRIPTION
**Pull request by jsonbruce**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

**Comment by jsonbruce** [comment]
Yes, You're absolutely right! 
I've completed the `gitk` manual. @agnivade

**Comment by jsonbruce** [comment]
Can I be an organization member of  `tldr pages`, So I can do more favor.  @agnivade

**Comment by agnivade** [comment]
Hey @jsonbruce, thanks for the interest !

We are in the process of formalizing a process of how to add more members in our org. There are a few guidelines that we follow which are mentioned here - https://github.com/tldr-pages/tldr/issues/1209. I would recommend that you take a look at these and familiarize yourself. Also have a look at our style guide. Then maybe you can start hanging around a bit more reviewing PRs and commenting on issues.

Once you start getting a hang of things and contribute regularly, we will be happy to add you to our org.

File: pages/linux/gitk.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# gitk
+
+> A graphical git repository browser.
+
+- Show repository browser in current git repository:
```
Comment (sbrl) [inline pages/linux/gitk.md]: Perhaps `Show the repository browser for the current git repository` would read better?

Comment (jsonbruce) [reply pages/linux/gitk.md]: Yes. You're right.
`Syntax Error`

**Comment by sbrl** [comment]
Thanks a ton, @jsonbruce! :D

**Comment by kevin91liu** [comment]
I see that `gitk.md` was intentionally added to `linux/`, presumably under the assumption that it's Linux-specific. I have `gitk` on my OSX machine. Perhaps it should be moved to `common/`, or copied to `osx/` if it isn't available on windows or sunos? 

See https://github.com/tldr-pages/tldr/issues/2018

**Pull request merged**